test/test_files/ctv_godunov_plm/ctv_godunov_plm.i

```diff
@@ -15,7 +15,6 @@ time.cfl              =   0.5        # CFL factor
 #.......................................#
 time.plot_interval  =  20   # Steps between plot files
 time.checkpoint_interval =   -1  # Steps between checkpoint files
-io.KE_int = 1        # calculate kinetic energy 
```

Comment:
Is there a regression text that exercises ke and enstrophy computation?

Reply:
yep, tgv_godunov does